spec/requests/categories/create_spec.rb

```diff
@@ -0,0 +1,36 @@
+RSpec.describe 'Category create', type: :request do
+  let(:category) { create(:category) }
+
+  context 'with valid params' do
+    let(:params) do
+      { category: { name: :name } }
```

Comment:
Why name is symbol? Why not like this?
```suggestion
      { category: { name: 'Category Name' } }
```